docs/index.md

```diff
@@ -67,6 +67,8 @@ $ prefab --output out/x86 --build-system cmake --platform android \
     --abi x86 --os-version 21 --ndk-version 21 --stl c++_shared deps
 $ prefab --output out/x86_64 --build-system cmake --platform android \
     --abi x86_64 --os-version 21 --ndk-version 21 --stl c++_shared deps
+$ prefab --output out/riscv64 --build-system cmake --platform android \
+    --abi riscv64 --os-version 35 --ndk-version 27 --stl c++_shared deps
```

Comment:
This doesn't match the text above the snippet. Update the text above to say r27 and add a note about why `--os-version` is different for the `riscv64` ABI?

Reply:
Updated to changed NDK version to 27 throughout the snippet and a note explaining the os-version difference.